MetaMask/metamask-extension
# Typography to text

## Explanation

Updating Typography to Text custom-spending-cap.js

Part of: #17670


## Screenshots/Screencaps


### Before

![Before](https://user-images.githubusercontent.com/86345323/228854170-f0eb0a87-ed06-4e19-b2b9-ab4de100fde4.png)

### After

![After](https://user-images.githubusercontent.com/86345323/228854273-59bf6035-d6d1-4d94-9ca4-b86b2782b1d5.png)

## Manual Testing Steps

- Go to the latest build of storybook on this PR
- Search custom-spending-cap.js in the side panel
- Check custom-spending-cap.js story agains develop storybook https://metamask.github.io/metamask-storybook/?path=/story/components-app-customspendingcap--default-story


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
